README.md

```diff
@@ -0,0 +1,27 @@
+# Roccat-tools
+
+## Add udev rules for your devices.
```

Comment:
none of this will be seen by users.

Reply:
Yes it's a pity. I included a link in the `metadata.xml` file, but still.

It seems other flatpak apps are in the same situation, for instance https://github.com/flathub/org.openrgb.OpenRGB explains how to install `udev.rules` in the README.md.

Is there a better solution ?

Also, as explained in the PR description, I added a dummy default desktop file. We might exec a command to display a message redirecting the user to the documentation ? (a gtk alert + link, a console message + link ?)

Reply:
See https://github.com/flathub/flathub/pull/4039/files#r1185821564